Source/dom/models/__tests__/CurvePoint.test.js

```diff
@@ -33,7 +33,7 @@ test('should be able to modify a CurvePoint', () => {
   expect(p.point.toJSON()).toEqual({ x: 0.3, y: 0.4 })
 })
 
-test('should show if a point is selected', () => {
+test('switching to path edit mode does not select a point (#32796)', () => {
```

Comment:
@acf I don't think this change is right. 

The way I see it, this test was describing the expectation that the `CurvePoint` model reflects the `selected` state correctly when a certain set of actions is performed on the document. Correct me if I'm wrong. 

Now the test does a completely different thing. Your changes turn the test into a verification of application behavior when entering Shape Edit mode, which to me does not belong in SketchAPI tests.

In a hypothetical scenario, let's say that in the future the backing store for the `isSelected` property changes and it stops being computed correctly. This test will still happily pass.

The problem was, until now, the test assumed a point is immediately selected when switching handler to `ShapeEventHandler` and this changed. 

In my opinion, the solution for this test is to change the set of actions that lead to having a point selected after switching to "path edit mode". Specifically, we should be able to access the `shapeEventHandler`, and call `selectNext()` on its `pathController`. Something like the following pseudocode assuming the CocoaScript API allows.

```
shapeEventHandler.pathController.selectNext()
```

After this, the expectation can stay `expect(shape.points[0].isSelected()).toBe(true)`.

Reply:
fair point.  :-)